subject_10/subject_10/CustomTableViewCell.swift

```diff
@@ -13,45 +13,50 @@ class CustomTableViewCell: UITableViewCell {
     //UI部品を作る
     private let leftLabel: UILabel = {
         let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.setContentHuggingPriority(UILayoutPriority(251), for: .horizontal)
+        label.textAlignment = .left
         return label
     }()
+    
     private let rightLabel: UILabel = {
         let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textAlignment = .right
+        label.textColor = .gray
         return label
     }()
-    
-    public func configure(_ prefectureName: String, _ indexNumber: String) {
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+
+        fatalError("not implemented")
+    }
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
         //UITableViewCellのcontentView: UIViewにメソッドaddSubViewを使って作ったUI部品を表示させる
         self.contentView.addSubview(leftLabel)
         self.contentView.addSubview(rightLabel)
-        
-        leftLabel.text = prefectureName
-        rightLabel.text = indexNumber + "番目の都道府県です"
-        leftLabel.textAlignment = .left
-        rightLabel.textAlignment = .right
-        //追加
-        rightLabel.textColor = UIColor.gray
+
+        NSLayoutConstraint.activate([
+            // horizontal
+            leftLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 20),
+            leftLabel.trailingAnchor.constraint(equalTo: rightLabel.leadingAnchor),
+            rightLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+
+            // vertical
+            leftLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 5),
+            leftLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -5),
+            rightLabel.topAnchor.constraint(equalTo: leftLabel.topAnchor),
+            rightLabel.bottomAnchor.constraint(equalTo: leftLabel.bottomAnchor),
+        ])
```

Comment:
frameでの指定には様々な問題があるので、制約ベースのレイアウトを使いましょう。

> In many ways, programmatically defining a view’s frame provides the most flexibility and power. When a change occurs, you can literally make any change you want. Yet because you must also manage all the changes yourself, laying out a simple user interface requires a considerable amount of effort to design, debug, and maintain. Creating a truly adaptive user interface increases the difficulty by an order of magnitude.

Auto Layout Guide: Understanding Auto Layout
https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/index.html